docs/resources/samples/getting-started.md

```diff
@@ -0,0 +1,744 @@
+---
+title: 'Getting started with Office Scripts'
```

Comment:
I know we're pulling information in, but I'm not sure we should include this article. It's too similar to the existing "Scripting Fundamentals" article and might confuse both customers and SEO.

Reply:
Tracking with [feature](https://office.visualstudio.com/OC/_workitems/edit/4924924)